crosstool/setup.bzl

```diff
@@ -0,0 +1,109 @@
+"""Configure the Apple CC toolchain"""
+
+load("//crosstool:osx_cc_configure.bzl", "configure_osx_toolchain")
+
+_DISABLE_ENV_VAR = "BAZEL_NO_APPLE_CPP_TOOLCHAIN"
```

Comment:
If anyone doesn't want to use the autodetected Apple C++ toolchain, they can just not pass all the custom `--*crosstool_top` flags. Is there still a need for this env var?

Reply:
Yea I only left it for debugging, I didn't put it in the docs. We could underscore it if we wanted it to look more private